### PR TITLE
CAW-1: Update dependencies

### PR DESCRIPTION
**Pull request by roblafeve**
## Description for [CAW-1](https://crema.atlassian.net/browse/CAW-1)

- Update dependencies to latest version
- Remove src/utils/decoratorCentered in favor of 3rd party addon
- Remove `EXTEND_ESLINT=true SKIP_PREFLIGHT_CHECK=true` from scripts
- Add husky-add-issue-tracker package

### 📋 Acceptance Criteria

> Checked off by the PR **Assignee(s)**

- [x] Scripts work as expected

### 👩‍🔬 Test Instructions

1. `npm ci`
2. `npm start` works ☑️
3. `npm run test:ci` works ☑️

## 🔎 Reviewer Checklist

> Checked off by the PR **Reviewers**

### Required

> These always need to be checked

- [x] Merge destination is correct
- [x] Code is correct as understood and conforms to quality standards
- [x] Tests have been added where appropriate (unit, visual, end-to-end)
- [x] Acceptance Criteria have been met

---

>### Roles & Responsibilities
>
>#### 👨‍💻 Assignee
>
>- Initiator of this PR (be sure to set in GitHub UI)
>- Addresses feedback and change requests
>- Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
>#### 👩‍💻 Reviewer
>
>- Invited to review PR by **Assignee** (via GitHub UI)
>- Is expected to complete a review and address followup

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,60 +3,60 @@
...
   },
   "scripts": {
-    "start": "EXTEND_ESLINT=true SKIP_PREFLIGHT_CHECK=true react-scripts start",
-    "start:ci": "EXTEND_ESLINT=true SKIP_PREFLIGHT_CHECK=true npm-run-all build static",
-    "build": "EXTEND_ESLINT=true SKIP_PREFLIGHT_CHECK=true react-scripts build",
-    "test": "EXTEND_ESLINT=true SKIP_PREFLIGHT_CHECK=true react-scripts test --testMatch **/**/test.{ts,tsx}",
+    "start": "react-scripts start",
+    "start:ci": "npm-run-all build static",
+    "build": "react-scripts build",
+    "test": "react-scripts test --testMatch \"**/?(*.)+(spec|test).[jt]s?(x)\"",
```
Comment (blaketarter) [inline package.json]: @roblafeve If we don't include `EXTEND_ESLINT=true` won't CRA just use its internal eslint config? [https://create-react-app.dev/docs/advanced-configuration](https://create-react-app.dev/docs/advanced-configuration)

Comment (roblafeve) [reply package.json]: I think you're right. I will have to look into this a bit more. I know there's an additional "experimental" way to do this without mucking up the scripts, but I'm not using that at the moment. Will look into this further.

File: package.json [hunk 2]
```diff
@@ -70,7 +70,8 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "npm run prepush"
+      "pre-push": "npm run prepush",
+      "prepare-commit-msg": "husky-add-issue-tracker"
```
Comment (blaketarter) [inline package.json:75]: 👏

Comment (roblafeve) [reply package.json:75]: Yeah, they have this on OrgAdmin—and I like it!

**Review by blaketarter — COMMENTED** [review]
Other than the removal of `EXTEND_ESLINT=true` this looks good to me.

File: package.json (hunk 1 above)
Comment (blaketarter) [reply package.json]: @roblafeve What's this experimental way you're talking about? 👀

Comment (roblafeve) [reply package.json]: @blaketarter https://create-react-app.dev/docs/setting-up-your-editor/#experimental-extending-the-eslint-config

Comment (roblafeve) [reply package.json]: @blaketarter I've added `EXTEND_ESLINT=true` back to those scripts. I misunderstood what that bit in the docs was saying—they are still required.

**Review by blaketarter — APPROVED** [review]

**Pull request merged**